packages/entities/entities-certificates/src/components/CertificateList.vue

```diff
@@ -2,13 +2,13 @@
   <div class="kong-ui-entities-certificates-list">
     <EntityBaseTable
       :cache-identifier="cacheIdentifier"
-      disable-pagination-page-jump
       :disable-sorting="disableSorting"
       :empty-state-options="emptyStateOptions"
       enable-entity-actions
       :error-message="errorMessage"
       :fetcher="fetcher"
       :fetcher-cache-key="fetcherCacheKey"
+      :hide-toolbar="hideTableToolbar"
```

Comment:
question: what criteria should be used to determine when the toolbar should be hidden?

Reply:
The toolbar is hidden when fetcher returns no records and there is no search query or filters. Something like a "true" empty state. In that case non of the toolbar items (search, column visibility menu) are actionable.